spec/processors/jpeg2k_spec.rb

```diff
@@ -19,6 +19,12 @@
 
   end
 
+  describe ".srgb_profile_path" do
+    it "exists" do
+      expect(File.exists?(described_class.srgb_profile_path)).to eq true
```

Comment:
Would this test have failed before your change to `Jpeg2kImage.srgb_profile_path`?